plugins/generator-1.18.2/forge-1.18.2/procedures/block_remove_drop.java.ftl

```diff
@@ -1,6 +1,7 @@
+<#include "mcelements.ftl">
 if(world instanceof Level) {
     Block.dropResources(world.getBlockState(
-        new BlockPos((int)${input$x},(int)${input$y},(int)${input$z})), (Level) world, new BlockPos((int)${input$x2},(int)${input$y2},(int)${input$z2}));
+        ${toBlockPos(input$x,input$y,input$z)}), (Level) world, ${toBlockPos(input$x2,input$y2,input$z2)});

```

Comment:
I would declare a BlockPos variable here and use it in all 3 cases since the case allows to do so.
Otherwise, the helper method usage looks OKay.